app/src/shared/components/Toolbar/index.jsx

```diff
@@ -1,49 +1,108 @@
 // @flow
```

Comment:
Toolbar file got rewritten using `styled-components`. New Toolbar (with the scroll sensitivity thing) is just 4 lines more than the previous one (incl. css). 💃